dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/TrackerImportParams.java

```diff
@@ -107,6 +107,13 @@
     @Builder.Default
     private ValidationMode validationMode = ValidationMode.FULL;
 
+    /**
+     * Should text pattern validation be skipped or not, default is not.
+     */
+    @JsonProperty
+    @Builder.Default
+    private boolean skipPatternValidation = false;
```

Comment:
As the other issue we  saw... this could be simplified by removing @Builder.Default and not assigning to false?